waiter/src/waiter/token.clj

```diff
@@ -719,3 +725,119 @@
                                              :status http-405-method-not-allowed})))
     (catch Exception ex
       (utils/exception->response ex req))))
+
+(defn make-index-event
+  [type object]
+  {:object object :type type})
+
+(defn start-tokens-watch-maintainer
+  [kv-store tokens-update-chan tokens-watch-channels-update-chan watch-refresh-timeout-ms exit-chan]
```

Comment:
We should try and avoid passing in channels and instead create the channels inside this function in the `let` block if possible.